### PR TITLE
Upgrade to LLVM 13/Julia 1.8

### PR DESCRIPTION
**Pull request by Gnimuc**

### CODE REVIEW

**Review by pemryan — APPROVED** [review]
trivial versioning updates.  I suggest this PR be merged  ASAP.

**Comment by vchuravy** [comment]
If we can I would love to be able to support multiple LLVM versions here as well.

**Comment by vchuravy** [comment]
@pemryan this PR right now breaks compatibility with Julia 1.7

File: Project.toml [hunk 1]
```diff
@@ -11,10 +11,10 @@ TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
 
 [compat]
 CEnum = "0.4"
-Clang_jll = "~12.0.1"
+Clang_jll = "~13.0.1"
```
Comment (vchuravy) [inline Project.toml:14]:
```suggestion
Clang_jll = "12,13"
```

File: Project.toml [hunk 2]
```diff
@@ -11,10 +11,10 @@ TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
...
 Downloads = "1.4"
 TOML = "1.0"
-julia = "~1.7.0"
+julia = "~1.8.0"
```
Comment (vchuravy) [inline Project.toml:17]:
```suggestion
julia = "1.7.0"
```

**Comment by Gnimuc** [comment]
The CI failure on macOS seems to be unrelated.

**Pull request merged**